Cocoapods Example/PortalSwift/ViewController.swift

```diff
@@ -204,6 +206,13 @@ class ViewController: UIViewController, UITextFieldDelegate {
       }
     }
   }
+    
+    @IBAction func handleEject(_: UIButton) {
+      print("Hello from the eject button")
+        self.PortalWrapper.eject(backupMethod: BackupMethods.iCloud.rawValue, user: self.user!) { result in
```

Comment:
does a user have to have backed up with icloud? maybe call this icloud eject instead

Reply:
Yeah exactly, in the example they have to backup with iCloud